daemon/create.go

```diff
@@ -14,26 +14,35 @@ import (
 	"github.com/opencontainers/runc/libcontainer/label"
 )
 
+// ContainerCreate is the parameter set to ContainerCreate()
+type ContainerCreate struct {
+	Name            string
+	Config          *runconfig.Config
+	HostConfig      *runconfig.HostConfig
+	AdjustCPUShares bool
+	FromBuilder     bool
```

Comment:
Struggling to think that one through, but I'm not sure that's necessarily the case. What about an API caller calling create and passing Volumes? That still has to be differentiated from the builder case, right?